admiral/pkg/clusters/handler.go

```diff
@@ -72,8 +72,9 @@ func getDestinationRule(host string, locality string, gtpTrafficPolicy *model.Tr
 	}
 	outlierDetection := &v1alpha32.OutlierDetection{
 		BaseEjectionTime:      &types.Duration{Seconds: 300},
-		Consecutive_5XxErrors: &types.UInt32Value{Value: uint32(10)},
+		ConsecutiveGatewayErrors: &types.UInt32Value{Value: uint32(10)},
```

Comment:
We can continue with Consecutive_5XxErrors as it will include ConsecutiveGatewayErrors. My bad in noticing just the description of ConsecutiveGatewayErrors. The description of Consecutive_5XxErrors does say that it includes error considered gateway errors.

Reply:
As discussed we can keep the default config to use `ConsecutiveGatewayErrors` until we expose these parameters via GTP.